Tensile/KernelWriter.py

```diff
@@ -274,7 +274,7 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
         kStr += self.globalReadIncrement(kernel, self.unrollIdx, \
             tensorParametersB)
       if self.enable["Wait"]:
-        kStr += self.wait(kernel, tensorParametersA, tensorParametersB, 0, -1, -1, "wait for global read")
+        kStr += self.wait(kernel, tensorParametersA, tensorParametersB, 0, -1, -1, "3wait for global read")
```

Comment:
What's the purpose of the numbers added to the comments?

Reply:
Debug - it is useful when looking at the assembler source to see which line in the KernelWriter generated the wait.